static/js/common/upload-addon.js

```diff
@@ -261,13 +262,99 @@
             });
 
 
+
+
+
+            $('#id_admin_override_validation').addClass('addon-upload-failure-dependant')
```

Comment:
Ugh. I'm pretty sure I'm the one who came up with that class name. But it should really be "depend**e**nt".